### PR TITLE
Moving towards AWX as the default

### PR DESCRIPTION
**Pull request by matburt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Ansible Tower/AWX Operator
```
Comment (geerlingguy) [inline README.md:1]: Maybe change title to 'Ansible AWX Operator' too to be clear? Or at least change order to "Ansible AWX/Tower Operator"

File: build/chain-operator-files.yml [hunk 2]
```diff
@@ -6,19 +6,19 @@
   gather_facts: false
 
   vars:
-    operator_image: ansible/tower-operator:0.4.0
+    operator_image: ansible/awx-operator:0.4.0
```
Comment (geerlingguy) [inline build/chain-operator-files.yml:9]: Just verifying; have you pushed this image?

File: deploy/awx-operator.yaml [hunk 3]
```diff
@@ -143,14 +143,14 @@ spec:
...
-  group: tower.ansible.com
+  group: awx.ansible.com
   names:
-    kind: Tower
-    listKind: TowerList
-    plural: towers
-    singular: tower
+    kind: AWX
+    listKind: AWXList
+    plural: awxs
```
Comment (geerlingguy) [inline deploy/awx-operator.yaml:153]: lol that is impossible to say :D

**Review by geerlingguy — COMMENTED** [review]
Looks good - one thing that is missing is any CI; at least for the interim, could you consider lifting the Travis CI config and using public OSS Travis so major changes like this can get tested? https://github.com/geerlingguy/tower-operator/blob/master/.travis.yml

**Comment by matburt** [comment]
I'm thinking I'm going to move this to a dedicated AWX operator. There's too much drift between AWX and Tower to maintain a unified operator consistently. We'll use this to get the AWX part nailed down and sort out a Tower operator afterwards.

**Comment by shanemcd** [comment]
I just added Travis to this repo, seems to be running now.

**Comment by nicolaibaralmueller** [comment]
When will this be merged?

**Pull request merged**